test/test.js

```diff
@@ -166,87 +161,26 @@ describe('cron-builder', function () {
 
     it('resets the value to the default "*" when removing the only value', function () {
         cron = new cb();
-        cron.set(['2020'], 'year');
-        cron.removeValue('2020', 'year');
-        expect(cron.get('year')).to.equal('*');
+        cron.set(['7'], 'minute');
+        cron.removeValue('7', 'minute');
```

Comment:
may want to have a check in here that the minute was actually set to 7 instead of skipped over and never actually changed from the default